label_studio/ml_models/models.py

```diff
@@ -6,6 +6,15 @@
 from django.utils.translation import gettext_lazy as _
 from ml_model_providers.models import ModelProviderConnection
 from projects.models import Project
+from rest_framework.exceptions import ValidationError
+
+def validate_string_list(value):
```

Comment:
Mind putting type definitions, and maybe a docstring here? It wasn't obvious what's supposed to be inside `value` to me.

Reply:
it is list of string which is being validated for type, incase the value sent is list of integers we will error out. I can add type as list